modules/surge-docs/src/test/java/javadocs/commandapp/BankAccountCommandModel.java

```diff
@@ -0,0 +1,84 @@
+// Copyright © 2017-2021 UKG Inc. <https://www.ukg.com>
+
+package javadocs.commandapp;
+
+import javadocs.commandapp.account.BankAccount;
+import javadocs.commandapp.command.BankAccountCommand;
+import javadocs.commandapp.command.CreateAccount;
+import javadocs.commandapp.command.CreditAccount;
+import javadocs.commandapp.command.DebitAccount;
+import javadocs.commandapp.event.BankAccountCreated;
+import javadocs.commandapp.event.BankAccountEvent;
+import javadocs.commandapp.event.BankAccountUpdated;
+import surge.javadsl.command.AggregateCommandModel;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+// #command_model_class
+public class BankAccountCommandModel implements AggregateCommandModel<BankAccount, BankAccountCommand, BankAccountEvent> {
+    // users can feel free to use vavr pattern matching as an alternative.
```

Comment:
can you move this at line 25 ... (so just before we use instanceof) 
And maybe change the text to "...as an alternative to instanceof"

Reply:
added